### PR TITLE
Removed definitions that were moved to common from node_agent.yaml

### PR DESCRIPTION
**Pull request by nnDarshan**
tendrl-bug-id:Tendrl/node-agent#311
Signed-off-by: nnDarshan <darshan.n.2024@gmail.com>

### CODE REVIEW

**Comment by nnDarshan** [comment]
@r0h4n @anmolbabu @shtripat @nthomas-redhat Please review

File: tendrl/node_agent/objects/definition/node_agent.yaml [hunk 1]
```diff
@@ -19,247 +19,6 @@ namespace.node_agent:
...
-      enabled: true
-      list: nodes/$Node_context.node_id/Disks/used
-    FreeDisk:
-      attrs:
-        disk_id:
-          help: "File system UUID"
-          type: String
-      enabled: true
-      list: nodes/$Node_context.node_id/Disks/free
     NodeNetwork:
```
Comment (shtripat) [inline tendrl/node_agent/objects/definition/node_agent.yaml]: Why not this? Shouldn't this be moved to commons as well?

File: tendrl/node_agent/objects/definition/node_agent.yaml [hunk 2]
```diff
@@ -443,183 +202,6 @@ namespace.node_agent:
...
-          type: String
-        status:
-          help: "Node status"
-          type: String
-
-      enabled: true
-      list: nodes/$Node_context.node_id/Node_context
-      value: nodes/$Node_context.node_id/Node_context
-      help: Node Context
     ClusterNodeContext:
```
Comment (shtripat) [inline tendrl/node_agent/objects/definition/node_agent.yaml]: Same as above as this is nothing specific to node-agent and commons can have this as well I feel.

File: tendrl/node_agent/objects/definition/node_agent.yaml (hunk 1 above)
Comment (nnDarshan) [inline tendrl/node_agent/objects/definition/node_agent.yaml]: @r0h4n @anmolbabu what do you feel ?

File: tendrl/node_agent/objects/definition/node_agent.yaml (hunk 2 above)
Comment (nnDarshan) [inline tendrl/node_agent/objects/definition/node_agent.yaml]: @r0h4n @anmolbabu what do you feel ?

Comment (anmolbabu) [inline tendrl/node_agent/objects/definition/node_agent.yaml]: I feel NodeContext, TendrlContext, Node and its associated atoms and any other atoms and objects that are required/used by all/more than one integration module need to move to commons, This is my understanding but I would like to hear from @r0h4n

Comment (anmolbabu) [inline tendrl/node_agent/objects/definition/node_agent.yaml]: And anything generic like OS and others also might find their best place in commons but others I have not much info about it so I think @r0h4n might be able to comment better...

Comment (anmolbabu) [inline tendrl/node_agent/objects/definition/node_agent.yaml]: But going by the logic of being general most(all) of these objects might need to move to commons but what I gathered from @r0h4n is that the shared objects move to commons @r0h4n Please correct me if I am wrong...

File: tendrl/node_agent/objects/definition/node_agent.yaml (hunk 1 above)
Comment (r0h4n) [inline tendrl/node_agent/objects/definition/node_agent.yaml]: Move to commons

File: tendrl/node_agent/objects/definition/node_agent.yaml (hunk 2 above)
Comment (r0h4n) [inline tendrl/node_agent/objects/definition/node_agent.yaml]: Move to commons

File: tendrl/node_agent/objects/definition/node_agent.yaml (hunk 1 above)
Comment (nnDarshan) [inline tendrl/node_agent/objects/definition/node_agent.yaml]: Okay, will move it

File: tendrl/node_agent/objects/definition/node_agent.yaml (hunk 2 above)
Comment (nnDarshan) [inline tendrl/node_agent/objects/definition/node_agent.yaml]: Okay, will move it

**Pull request merged**